.github/workflows/release.yml

```diff
@@ -0,0 +1,103 @@
+name: Build Releases
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - 'feature/issue-57_standalone-executable'
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+            os: [ubuntu-latest, windows-latest, macos-latest]
+
+    steps:
+      - name: "Create base filename for all artifacts"
+        id: basefn
+        shell: bash
+        run: |
+          FILEPATH=$(echo rctab_${{ github.ref_name }}_${{ runner.os }} | sed -e 's/\//_/g')
+          echo "FILEPATH=$FILEPATH" >> $GITHUB_OUTPUT
+
+      # Normalize platform-specific filepaths generated by gradle
+      - name: "Create .zip filename"
+        id: zipfn
+        shell: bash
+        run: echo "FILEPATH=build/${{ steps.basefn.outputs.FILEPATH }}.zip" >> $GITHUB_OUTPUT
+      - name: "Get extension"
+        id: ext
+        shell: bash
+        run: |
+          if [ ${{ runner.os }} == 'Windows' ]; then
+            echo "EXT=.exe" >> $GITHUB_OUTPUT
+          elif [ ${{ runner.os }} == 'Linux' ]; then
+            echo "EXT=" >> $GITHUB_OUTPUT
```

Comment:
In my failed Linux check today, the lack of extension was a little confusing, but the OS seemed to at least recognize it was a package when I tried to run it (and rejected it due to security policy, as mentioned). Hopefully it's correct to keep it without an extension (e.g. no `.deb`). Will leave it up to your Virtualbox testing to verify.

Reply:
I've corrected this: previously, I just shipped the binary (thus no extension), but changed it to use .deb now and forgot to sync that change.

The .deb seems to be correct -- however, I haven't booted my linux partition in several years and it seems like testing on my end will be pretty hard. The machine has no access to the internet (seems like the wifi hardware is too new), and when manually copying the .deb via a USB stick, it's unable to open it because it uses a newer compression format not supported by Ubuntu 16.04.

All this to say: if somebody else has faster access to Linux, could you test it fully? Otherwise, I'll need to install virtualbox or reformat my linux partition, both of which will take some time to set up.

Reply:
Ah, that does remind me-- I've got some Raspberry Pis ready-to-go laying around. I'll see if I can boot one of those up.

Reply:
i've reinstalled a more recent Ubuntu and confirmed that this works, though the install doesn't put the binary in the $PATH -- you have to manually launch it from `/opt/`. I can't figure out how to make the package more easily launchable, so I'm happy to call this "good enough" for now given the lack of linux users, and the easy workaround. I've added info to the README. We can revisit later if we want to make this simpler, but for now, just run `/opt/rctab/bin/RCTab` to launch after installing the .deb.